src/lib.rs

```diff
@@ -200,6 +201,7 @@ pub fn ipc_irq_handler() {
 #[derive(Debug, Copy, Clone)]
 pub struct SystemMode {
     pub lte_support: bool,
+    pub lte_psm_support: bool,
```

Comment:
I don't love this here, but for now this is probably fine. I can image there there'll be more options like this in the future and then it'll have to be refactored

Reply:
Yes I know this is not the best place, as I said I tried first adding it to creating LteLink but as it is used in other places that was not possible.
Maybe a separate struct like a LteMode with would be Option<> in SystemMode (so when you don't have it lte is disabled, when you have it it is enabled with the right settings?